.changeset/wild-goats-march.md

```diff
@@ -0,0 +1,20 @@
+---
+'@itwin/itwinui-react': minor
+'@itwin/itwinui-css': minor
+---
+
+Added a new FileUploadCard component which serves as a default UI for when a single file is uploaded. This can also be used as a child of FileUpload
+
+```jsx
+<FileUploadCard />
+```
+
+```jsx
+const [files, setFiles] = React.useState<File[]>([]);
+
+<FileUpload
+  onFileDropped={console.log()}
```

Comment:
would be better to show it updating state rather than console.log

Reply:
This has been changed